OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -152,7 +154,7 @@ void INotifyOwnerChanged.OnOwnerChanged(Actor self, Player oldOwner, Player newO
 		{
 			ClearQueue();
 
-			playerPower = newOwner.PlayerActor.Trait<PowerManager>();
+			playerPower = newOwner.PlayerActor.TraitOrDefault<PowerManager>();
```

Comment:
... otherwise you will also need to update `playerActor` to the new owner here (but i'd prefer we simply didn't define that).

Reply:
So.. do i need to change this here?